src/platform/inc/ManagementModule.h

```diff
@@ -4,6 +4,9 @@
 #ifndef MANAGEMENTMODULE_H
 #define MANAGEMENTMODULE_H
 
+#include <Mmi.h>
+#include <rapidjson/document.h>
```

Comment:
Remove these. Both are included from PlatformCommon.h already. There shall be no headers included from this header file.

Reply:
Do not include headers from headers like this. This was already done properly, please remove this line.